bindata/linuxptp/ptp-daemon.yaml

```diff
@@ -127,7 +127,12 @@ spec:
             secretName: linuxptp-daemon-secret
         {{ if (eq .EnableEventPublisher true) }}
         - name: pubsubstore
+          {{ if (hasPrefix "http://" .EventTransportHost) }}
```

Comment:
has prefix http:// check might be an issue, also transport host can be given without http , And we need to support fo https .The best check will be to check for NOT amqp

Reply:
updated